src/Resources/contao/library/Contao/Widget.php

```diff
@@ -381,11 +381,11 @@ public function __get($strKey)
 
 			case 'value':
 				// Encrypt the value
-				if ($this->arrConfiguration['encrypt'])
+				if (isset($this->arrConfiguration['encrypt']) && true === $this->arrConfiguration['encrypt'])
```

Comment:
Isn't `isset($this->arrConfiguration['encrypt'])` also true for `$this->arrConfiguration['encrypt'] = false;`?
Therefore this seems to be required IMO.